spec/features/post_peep_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require_relative './helpers/sign_up.rb'
+require_relative './helpers/add_peep.rb'
+
+feature "User adds a new peep" do
+
+  scenario "must be logged in send a peep" do
+    visit '/peeps'
+    expect(page).not_to have_content "Add peep"
+  end
+
+  scenario "1 new peep can be added" do
+    sign_up
+    add_peep "this is a short peep"
+    expect(page).to have_css('.peep', :count => 1)
+  end
+
+   scenario "with the intended peep message showing" do
+    sign_up
+    add_peep "this is a short peep"
+    expect(page).to have_css('.peep', :text => "this is a short peep")
+  end
+
+  scenario "peeps are ordered chronologically" do
+    sign_up
+    add_peep "this is a short peep"
+    add_peep "this is a second short peep"
+    expect(page).to have_css('li:nth-child(1)', :text => "this is a short peep")
```

Comment:
great that you're mastering these css selectors, but I'd be careful about your acceptance tests too tightly locked to your CSS layout - if that changes this test will break.  You'd probably be better off just checking for the presence in the page and their relative ordering rather than being in a particular css selector